src/app/core/basic-datatypes/discrete/discrete-import-config/discrete-import-config.component.ts

```diff
@@ -54,23 +56,30 @@ export class DiscreteImportConfigComponent implements OnInit {
   }
 
   private getFormValues(additional: any) {
-    if (!additional) {
-      additional = {};
-    }
+    additional = additional || {};
+
+    const enumEntity = this.configurableEnumService.getEnum(
+      this.schema?.additional,
+    );
```

Comment:
we probably shouldn't do this if schema.additional is undefined. Looking at the code, this might create a new enum entity. And we do use the same import-config component for boolean and enums, so this may be a valid scenario to not have an `additional`